### PR TITLE
[SDL-4231] Remove references to library source, changed DFP version for ml-pic-iot-imu-sensiml-template project

### PR DESCRIPTION
**Pull request by mbruner63**
Remove references to library source, changed DFP version for ml-pic-iot-imu-sensiml-template project

### CODE REVIEW

File: firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/private/private.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
             <file>file:/C:/Users/mbruner/Documents/microchip/ml-pic-iot-imu-sensiml-template/firmware/ml-pic-iot-imu-sensiml-template.X/mcc_generated_files/pin_manager.c</file>
             <file>file:/C:/Users/mbruner/Documents/microchip/ml-pic-iot-imu-sensiml-template/firmware/ml-pic-iot-imu-sensiml-template.X/main.c</file>
             <file>file:/C:/Users/mbruner/Documents/microchip/ml-pic-iot-imu-sensiml-template/firmware/knowledgepack/knowledgepack_project/app_config.h</file>
+            <file>file:/C:/Users/mbruner/Documents/microchip/ml-pic-iot-imu-sensiml-template/firmware/knowledgepack/knowledgepack_project/sml_recognition_run.c</file>
```
Comment (cdknorow) [inline firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/private/private.xml]: This looks specific to you.

File: firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/configurations.xml.bak.bak [hunk 2]
```diff
@@ -0,0 +1,698 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configurationDescriptor version="65">
```
Comment (cdknorow) [inline firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/configurations.xml.bak.bak]: is this a backup file?

File: firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/configurations.xml.bak [hunk 3]
```diff
@@ -0,0 +1,644 @@
...
+            overriding="false">
+      </item>
+      <item path="../knowledgepack/sensiml/lib/calc_area.c"
+            ex="true"
+            overriding="false">
+      </item>
+      <item path="../knowledgepack/sensiml/lib/column_to_row_complex.c"
+            ex="true"
+            overriding="false">
+      </item>
```
Comment (cdknorow) [inline firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/configurations.xml.bak]: is this a backup file

**Review by cdknorow — APPROVED** [review]

File: firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/private/private.xml (hunk 1 above)
Comment (mbruner63) [reply firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/private/private.xml]: Removed File

File: firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/configurations.xml.bak.bak (hunk 2 above)
Comment (mbruner63) [reply firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/configurations.xml.bak.bak]: Removed file

File: firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/configurations.xml.bak (hunk 3 above)
Comment (mbruner63) [reply firmware/ml-pic-iot-imu-sensiml-template.X/nbproject/configurations.xml.bak]: removed file

**Pull request merged**